src/mod/externals/SmartPoint/Components/PlayerPrefsProvider.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "externals/il2cpp-api.h"
+
+// Forward declaration to avoid circular dependencies
+struct ViewerSettings;
+
+namespace SmartPoint::Components {
+    template <typename T>
+    struct PlayerPrefsProvider : ILClass<T> {
+        struct Fields {
+            bool _loaded;
+        };
+
+        static inline StaticILMethod<0x04c90330, ViewerSettings>
```

Comment:
This should return PlayerWork, not ViewerSettings, since that's what this get_instance's type is